Fix build on PostgreSQL 16, needed for va_list in a PostgreSQL header.

Index: psqlodbc.h
--- psqlodbc.h.orig
+++ psqlodbc.h
@@ -18,6 +18,7 @@
 #include <windows.h>
 #endif
 
+#include <stdarg.h>
 #include <stdio.h>				/* for FILE* pointers: see GLOBAL_VALUES */
 #ifdef POSIX_MULTITHREAD_SUPPORT
 #include <pthread.h>
